techniques/general/G139.html

```diff
@@ -2,155 +2,126 @@
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques.html">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Content that accepts user data input, with restrictions on the format, value, and/or
-               type of the input.
+            <h2>適用 (対象)</h2>
+            <p>利用者のデータ入力を受け付け、そのフォーマット、値及び／又は入力の種類に制限があるコンテンツ
             </p>
-            <p>This technique relates to:</p>
+            <p>これは、次の達成基準に関連する達成方法である:</p>
             <ul>
-               <li><a href="https://w3c.github.io/wcag/understanding/error-identification">Success Criterion 3.3.1: Error Identification</a> (Advisory)
+               <li><a href="https://w3c.github.io/wcag/understanding/error-identification">達成基準 3.3.1: エラーの特定</a> (Advisory)
                </li>
-               <li><a href="https://w3c.github.io/wcag/understanding/error-suggestion">Success Criterion 3.3.3: Error Suggestion</a> (Advisory)
+               <li><a href="https://w3c.github.io/wcag/understanding/error-suggestion">達成基準 3.3.3: エラー修正の提案</a> (Advisory)
                </li>
             </ul>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to help users find input errors where the information
-               supplied by the user is not accepted. This includes fields with missing required information
-               and fields with incorrect information. When users enter data input that is checked,
-               and input errors are detected, a link to that error is provided so that the user does
-               not have to search for it. One approach is to use server-side validation, and to re-display
-               the form (including any previously entered data), and a text description at the top
-               of the page that indicates the fact that there was an input error, describes the nature
-               of the problem, and provides a link the field(s) with a problem.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、利用者によって提供された情報が受け付けられない場合に、入力エラーの修正を支援することである。これには、必須入力欄が空欄の場合と、入力欄に正しくない情報が入力された場合とが含まれる。利用者による入力データをバリデートした結果、入力エラーが検出された場合に、この入力エラーの内容と場所に関する情報をテキストで提供することによって、利用者は問題のある箇所を特定することができる。一つのアプローチとして、サーバーサイドでのバリデーションを用いて、フォームを再表示し (すでに入力されたデータも含む)、ページの上部にテキストでの説明文を表示して、エラーがあったことを知らせるとともに問題の内容を説明して、利用者が問題のあった項目を簡単に見つけられるような方法を提供することがある。
```

Comment:
```suggestion
            <p>この達成方法の目的は、利用者によって提供された情報が受け付けられない場合に、入力エラーの修正を支援することである。これには、必須の情報が欠落しているフィールドと、正しくない情報のあるフィールドが含まれる。利用者がチェックされたデータ入力を入力して、入力エラーが検出された場合、そのエラーへのリンクを提供することによって、利用者はエラーを検索する必要はない。一つのアプローチとして、サーバーサイドでのバリデーションを用いて、フォームを再表示し (すでに入力されたデータも含む)、ページの上部にテキストでの説明文を表示して、問題の性質を説明し、問題のあったフィールドへのリンクを提供する。
```

Reply:
```suggestion
            <p>この達成方法の目的は、利用者によって提供された情報が受け付けられない場合に、入力エラーの修正を支援することである。これには、必須の情報が欠落しているフィールドと、正しくない情報のあるフィールドが含まれる。利用者がデータ入力を入力し、チェックされ、入力エラーが検出された場合、そのエラーへのリンクを提供することによって、利用者はエラーを探す必要がない。一つのアプローチとして、サーバーサイドでのバリデーションを用いて、フォームを再表示し (すでに入力されたデータも含む)、ページの上部にテキストでの説明文を表示して、問題の性質を説明し、問題のあったフィールドへのリンクを提供する。
```

Reply:
When users enter data input [that is checked], ... とやはりひとかたまりに捉えるべきですので、
> 利用者がデータ入力を入力し、チェックされ、

とはならないかなあと。data inputがチェックされるわけですので、意訳気味に「チェック対象の」としてみましたがいかがでしょう。あと、so that構文も素直に取ってみました。

```suggestion
            <p>この達成方法の目的は、利用者によって提供された情報が受け付けられない場合に、入力エラーの修正を支援することである。これには、必須の情報が欠落しているフィールドと、正しくない情報のあるフィールドが含まれる。利用者がチェック対象のデータ入力を入力し、入力エラーが検出された場合、利用者がそのエラーを探す必要がないように、エラーへのリンクを提供する。一つのアプローチとして、サーバーサイドでのバリデーションを用いて、フォームを再表示し (すでに入力されたデータも含む)、ページの上部にテキストでの説明文を表示して、問題の性質を説明し、問題のあったフィールドへのリンクを提供する。
```

Reply:
ありがとうございます。問題ないと思います！